examples/va/nfv/sriov/kustomization.yaml

```diff
@@ -0,0 +1,38 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+components:
+  - ../../../../va/nfv/sriov/
+  # - https://github.com/openstack-k8s-operators/architecture/va/hci?ref=main
```

Comment:
The URL should be `https://github.com/openstack-k8s-operators/architecture/va/nfv/sriov?ref=main`

Alternatively, comments on line 6 to 8 should be removed.

Reply:
Fixed